### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by cytadela8**
This adds CI. Currently tests are failing, but we will get some linter results.

### CODE REVIEW

**Comment by cytadela8** [comment]
I see we unnecesserly get both push and pull_request executions...

File: make_plan.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python2
```
Comment (krzys-h) [inline make_plan.py]: uhh add this to the TODO list :P

File: .github/workflows/pythontest.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Django check
+      run: ./manage.py check
+    - name: Django migrations
+      run: ./manage.py makemigrations --check --dry-run
+    # Tests are completelly broken
```
Comment (krzys-h) [inline .github/workflows/pythontest.yml]: add this to the TODO list too :P

Comment (cytadela8) [reply .github/workflows/pythontest.yml]: #35

File: make_plan.py (hunk 1 above)
Comment (cytadela8) [reply make_plan.py]: #129

**Comment by krzys-h** [comment]
`push` and `pull_request` are slightly different - `pull_request` probably takes into account possible automatic merge between the pull request branch and target branch (I'm not sure this is how it works with GitHub actions, but this is how it worked with GitHub pull request plugins for Jenkins)

Maybe you can set it up for pushes on master and all pull_requests instead? Or just pull_requests, rather than normal commits

**Comment by cytadela8** [comment]
@krzys-h I was hoping GH would send email when tests fail, but it doesn't so I can disable tests on other branches then master. Also GH limits might be kind of a concern. There is a 1000min/month/user limit.

**Comment by radrow** [comment]
#54 – is it fixed?

**Pull request merged**